WordPress/Classes/ViewRelated/Me/App Settings/About/AboutViewController.swift

```diff
@@ -217,49 +262,123 @@ open class AboutViewController: UITableViewController {
         #endif
     }()
 
+    private lazy var sharePresenter: ShareAppContentPresenter? = {
+        guard isRecommendAppSectionEnabled else {
+            return nil
+        }
+
+        let account = AccountService(managedObjectContext: ContextManager.shared.mainContext).defaultWordPressComAccount()
+        let presenter = ShareAppContentPresenter(account: account)
+        presenter.delegate = self
+
+        return presenter
+    }()
+
+    /// Convenience property to determine whether the recomend app section should be displayed or not.
+    private var isRecommendAppSectionEnabled: Bool {
+        return FeatureFlag.recommendAppToOthers.enabled && !AppConfiguration.isJetpack
```

Comment:
🤔  should we enable the share button feature for Jetpack app?

Reply:
Let's wait and release for WordPress first. Then if we need to iterate on anything we can do that and _then_ enable Jetpack.

Reply:
Sounds good! 👍